### PR TITLE
Remove deprecated SQL Database editions

### PR DESCRIPTION
**Pull request by MCKLMT**
### Changelog

* Remove old SQL Database editions (Web, Business)

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@MCKLMT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mimott). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: 101-mobile-app-create/azuredeploy.json [hunk 1]
```diff
@@ -36,7 +36,12 @@
       "defaultValue": "Basic",
       "metadata": {
         "description": "The type of database to create. The available options are: Web, Business, Basic, Standard, and Premium."
-      }
+      },
```
Comment (bmoore-msft) [inline 101-mobile-app-create/azuredeploy.json]: this list doesn't match the description - can we just move this to a variable and pick one?

File: 201-web-app-vm-dsc/azuredeploy.json [hunk 2]
```diff
@@ -105,13 +105,11 @@
     },
     "databaseEdition": {
       "type": "string",
-      "defaultValue": "Web",
+      "defaultValue": "Basic",
```
Comment (bmoore-msft) [inline 201-web-app-vm-dsc/azuredeploy.json]: ditto for allowed values, can we just pick one and use a var?

File: sonarqube-azuresql/azuredeploy.json [hunk 3]
```diff
@@ -67,13 +65,11 @@
       "defaultValue": "Basic",
       "allowedValues": [
         "Basic",
-        "Business",
-        "Premium",
         "Standard",
-        "Web"
+        "Premium"
```
Comment (bmoore-msft) [inline sonarqube-azuresql/azuredeploy.json]: move to var?

File: sonarqube-azuresql/nested/azureDBDeploy.json [hunk 4]
```diff
@@ -24,27 +24,22 @@
       "defaultValue": "Basic",
       "allowedValues": [
         "Basic",
-        "Business",
-        "Premium",
         "Standard",
-        "Web"
+        "Premium"
```
Comment (bmoore-msft) [inline sonarqube-azuresql/nested/azureDBDeploy.json]: move to var?

File: sqldb-django-on-ubuntu/azuredeploy.json [hunk 5]
```diff
@@ -75,9 +75,14 @@
     "edition": {
       "type": "string",
       "metadata": {
-        "description": "SQL Edition"
+        "description": "SQL Database Edition"
       },
-      "defaultValue": "Standard"
+      "defaultValue": "Standard",
```
Comment (bmoore-msft) [inline sqldb-django-on-ubuntu/azuredeploy.json]: move to var

File: sonarqube-azuresql/azuredeploy.json (hunk 3 above)
Comment (MCKLMT) [inline sonarqube-azuresql/azuredeploy.json]: fixed

File: sonarqube-azuresql/nested/azureDBDeploy.json (hunk 4 above)
Comment (MCKLMT) [inline sonarqube-azuresql/nested/azureDBDeploy.json]: fixed

File: 201-web-app-vm-dsc/azuredeploy.parameters.json [hunk 6]
```diff
@@ -41,6 +41,9 @@
     "databaseName": {
       "value": "cawadb07"
     },
+    "databaseEdition":{
+      "value": "Basic"
+    }
```
Comment (bmoore-msft) [inline 201-web-app-vm-dsc/azuredeploy.parameters.json]: missing comma

Comment (MCKLMT) [inline 201-web-app-vm-dsc/azuredeploy.parameters.json]: fixed

**Pull request merged**